src/lwc/gsChecklist/gsChecklist.js

```diff
@@ -0,0 +1,16 @@
+import { LightningElement, api, track } from 'lwc';
+import gsChecklistArrowRightAlterText from '@salesforce/label/c.gsChecklistArrowRightAlterText';
```

Comment:
I think Beth mentioned this earlier. Use "Alt" instead of "Alter" for the custom label and var name here.